crates/services/p2p/src/request_response/messages.rs

```diff
@@ -32,14 +34,77 @@ pub enum RequestMessage {
     TxPoolFullTransactions(Vec<TxId>),
 }
 
+// TODO: Do we want explicit status codes or an Error type?
```

Comment:
I think it's not a big problem to have more detailed error (with specific block id or thing like that) as soon as we add a limit on the size of the error.